### PR TITLE
Change error statement colour from black to red in patient creation form

### LINKED ISSUE #4867: Change error statement colour from black to red in patient creation form
**Describe the bug**

in the patient creation form, the error statement for mandatory fields are shown in black color making it hard to understand the statement

**To Reproduce**

https://care.coronasafe.in/facility/4adaaff2-c6b2-4530-a518-3fbe44752421/patient

**Expected behavior**

 the error statement should be shown in red

**Screenshots**

image

### PR DESCRIPTION
**Pull request by siva-balan**
## Proposed Changes

- Fixes #4867 
- Changes:
image
image

- Changed text field to text form field and added red color to errors in the select field.
@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Patient/PatientRegister.tsx [hunk 1]
```diff
@@ -537,7 +549,9 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             state.form.nationality === "India" &&
             !Number(state.form[field])
           ) {
-            errors[field] = "Please select ward";
+            errors[field] = (
+              <span className="text-red-500">Please select ward</span>
```
Comment (rithviknishad) [inline src/Components/Patient/PatientRegister.tsx:553]: @siva-balan wouldn't the FormField component apply the colors automatically?

You can make use of `SelectFormField` for gender select.

```suggestion
              <span className="text-red-500">Please select ward</span>
```

```suggestion
              Please select ward
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by siva-balan** [comment]
@rithviknishad I am unable to use SelectFormField for the States and districts. I am getting some errors, can you help with that... I am able to use that for gender but not for others.

**Comment by rithviknishad** [comment]
Hey @siva-balan 👋🏼 sure, happy to help you out.

Have a look at [`src/Components/Users/UserAdd.tsx`](https://github.com/coronasafe/care_fe/blob/develop/src/Components/Users/UserAdd.tsx#L887-L921) where similar State and District select menus were implemented using `SelectFormField`.

Let me know if you are still facing issues after looking at this.

image

**Comment by siva-balan** [comment]
@rithviknishad I already had a look at that, and I tried to implement it similarly but I am not able to click the states in the drop-down. Could you take a look at this?
0d62c76b-532c-4814-9e17-70aeecaf40bb
e385ad7e-9c8d-4626-98ea-777363133cfb

Tell me if there's anything I need to change.. Thanks for the help.

**Comment by rithviknishad** [comment]
@siva-balan I'm guessing that you are using `handleChange` which is for old components and wouldn't work with `FormField` components. For `FormField` components you shall use `handleFormFieldChange`.

**Comment by rithviknishad** [comment]
@siva-balan once there are no more legacy input fields that's using `handleChange`, you may remove that old `handleChange` altogether.

**Comment by siva-balan** [comment]
@rithviknishad Thanks for the fix, I was able to implement them, but the drop-down options are not fully visible due to using the select form field. They are being hidden by the below component. Any idea on how to solve that?
image
In the case of localbody, only one option is visible.

**Comment by rithviknishad** [comment]
@siva-balan this is probably because the parent container has `overflow-clip` class.

**Comment by rithviknishad** [comment]
@siva-balan the Material UI's `Card` component can be replaced with a `div` if `Card` is causing the clipping.

You may remove these if so:

image

**Pull request closed without merging**